### PR TITLE
add -L to test and test-cov

### PR DESCRIPTION
**Pull request by johnbrett**
Adding -L to all Makefile options as mentioned in https://github.com/hapijs/shot/issues/21

### CODE REVIEW

**Comment by johnbrett** [comment]
Replaced Makefile with npm scripts also https://github.com/hapijs/shot/issues/21#issuecomment-76947486 Thanks @AdriVanHoudt @lloydbenson

**Comment by AdriVanHoudt** [comment]
LGTM

**Comment by AdriVanHoudt** [comment]
This with #25 closes #21 right?

**Comment by hueniverse** [comment]
@arb can you review and merge?

File: package.json [hunk 1]
```diff
@@ -21,7 +21,9 @@
     "code": "1.x.x"
   },
   "scripts": {
-    "test": "make test-cov"
+    "test": "node_modules/lab/bin/lab -a code -L",
+    "test-cov": "node_modules/lab/bin/lab -a code -t 100 -L",
+    "test-cov-html": "node_modules/lab/bin/lab -a code -r html -o coverage.html -L"
```
Comment (chrisdickinson) [inline package.json]: it should be unnecessary to add `node_modules/lab/bin/` here – `node_modules/.bin/` is automatically on path.

Comment (chrisdickinson) [inline package.json]: will fix this in merge.

**Comment by chrisdickinson** [comment]
Merged in d30209e7b2c1fd52ebc29d6b05c42f87b7888a4a, published to npm as v1.4.2.

**Comment by lloydbenson** [comment]
Thanks Chris!

**Pull request closed without merging**